WebApplication/ClientApp/src/components/modalDownloadProgress.js

```diff
@@ -69,7 +69,7 @@ export class ModalDownloadProgress extends Component {
                         onAutostart={(downloadHyperlink) => {
                             downloadHyperlink.click();
                         }}
-                        onUrlClick={this.props.onClose}
+                        /* onUrlClick={this.props.onClose} */ // onClose in onUrlClick colides with onAutostart, causing the dialog to close itself when download starts. But we dont want it to close itself.
```

Comment:
What we can do about it?

Reply:
> causing the dialog to close itself when download starts. But we dont want it to close itself.

Oh... I thought we *want* to close dialog if download is autostarted, so I changed this code last week. Am I missed some discussion or mixed it up with something else?
@mivasi Could you comment?

Reply:
I think we don't want to close even if download starts.